docs/ums/README.md

```diff
@@ -302,11 +302,27 @@ ums.signOut('Elon Musk');
 
 </CodeSwitcher>
 
+## Current user
+When logged in, we get also the current logged-in user.  
+
+<CodeSwitcher :languages="{js:'JavaScript',py:'Python',bash:'cURL'}">
+<template v-slot:js>
+
+``` js
+const currentUser = ums.currentUser;
```

Comment:
Is it object or array or what is in this variable?

Reply:
It's a user object, the same as `ums.fetchUser()` will gives you.

Reply:
Shall we add some comments maybe?